components/phenotype-suggest-service/api/src/main/java/org/phenotips/solr/PhenotypeSuggestService.java

```diff
@@ -54,18 +57,19 @@
 @Component
```

Comment:
Every component must implement a role, but this doesn't implement anything, so it will fail to be instantiated.